README.example.md

```diff
@@ -65,7 +65,9 @@ Any env var you specify in such a file will override the configuration for the c
 
 #### Database connection
 
-You can set `DATABASE_URL` in `.env.local`, if for instance you use Docker for Postgres. `DATABASE_URL="postgresql://localhost:5432"`
+You can set `DATABASE_URL` in `.env.local`, if you for instance use Docker for Postgres: `DATABASE_URL="postgresql://localhost:5432"`
+Or if you just use a local postgres instance:
+`DATABASE_URL=postgresql://user:pass@localhost:5432`
```

Comment:
nice 💪 
looks a little off when rendered... might be worth it to throw some double spaces in there... but not too important 🤷‍♀️

Reply:
I think it looks fine, for now at least